.github/workflows/update-metadata-glsa.yml

```diff
@@ -0,0 +1,58 @@
+name: Keep GLSA metadata updated
+on:
+  schedule:
+    - cron:  '0 7 1 * 1'
+  workflow_dispatch:
+
+jobs:
+  keep-glsa-metadata-updated:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout portage-stable
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: ./portage-stable
+      - name: Checkout Gentoo
+        uses: actions/checkout@v3
+        with:
+          repository: gentoo-mirror/gentoo
+          path: gentoo
+          fetch-depth: 1
+          ref: master
+      - name: Checkout build scripts
+        uses: actions/checkout@v3
+        with:
+          repository: flatcar/flatcar-build-scripts
+          path: flatcar-build-scripts
+      - name: Update GLSA metadata
+        id: update-glsa-metadata
+        run: |
+          git config --global user.name "Flatcar Buildbot"
+          git config --global user.email "buildbot@flatcar-linux.org"
+          old_head=$(git -C portage-stable rev-parse HEAD)
+          # need to rsync binary data Manifest* and timestamp* which are not in the Git repos.
+          rsync --archive rsync://rsync.gentoo.org/gentoo-portage/metadata/glsa/{Manifest,timestamp}* ./gentoo/metadata/glsa/
+          cd portage-stable
+          GENTOO_REPO=../gentoo ../flatcar-build-scripts/sync-with-gentoo "metadata/glsa"
+          cd ..
+          new_head=$(git -C portage-stable rev-parse HEAD)
+          updated=0
+          if [[ "${new_head}" != "${old_head}" ]]; then
+              updated=1
+          fi
+          todaydate=$(date +%Y-%m-%d)
+          echo "UPDATED=${updated}" >>"${GITHUB_OUTPUT}"
+          echo "TODAYDATE=${todaydate}" >>"${GITHUB_OUTPUT}"
+      - name: Create pull request for main branch
+        uses: peter-evans/create-pull-request@v4
+        if: steps.update-glsa-metadata.outputs.UPDATED == 1
```

Comment:
i found the following in the action documentation:

> How the action behaves:
> 
> If there are changes (i.e. a diff exists with the checked-out base branch), the changes will be pushed to a new branch and a pull request created.
If there are no changes (i.e. no diff exists with the checked-out base branch), no pull request will be created and the action exits silently.
If a pull request already exists it will be updated if necessary. Local changes in the Actions workspace, or changes on the base branch, can cause an update. If no update is required the action exits silently.
If a pull request exists and new changes on the base branch make the pull request unnecessary (i.e. there is no longer a diff between the pull request branch and the base), the pull request is automatically closed. Additionally, if delete-branch is set to true the branch will be deleted.

So we could use these properties and get rid of the `UPDATED` checks/rev-parsing.

Reply:
removed UPDATED